tutorials/math/normalDist.py

```diff
@@ -0,0 +1,95 @@
+## \file
+## \ingroup tutorial_math
+## \notebook
+## Tutorial illustrating the new statistical distributions functions (pdf, cdf and quantile)
+##
+## \author Anna Kreshuk
```

Comment:
I don't think this will work in our doc. Perhaps Anna can be cited and your name is the only one to have as \author?